google-cloud-storage/src/main/java/com/google/cloud/storage/GrpcStorageOptions.java

```diff
@@ -236,6 +266,22 @@ public Builder setTerminationAwaitDuration(Duration terminationAwaitDuration) {
       return this;
     }
 
+    /**
+     * Option which signifies the client should attempt to connect to gcs via Direct Path.
+     *
+     * <p>In order to use direct path, both this option must be true and the environment variable
+     * (not system property) {@code GOOGLE_CLOUD_ENABLE_DIRECT_PATH_XDS} must be true.
+     *
+     * <p><i>NOTE</i>There is no need to specify a new endpoint via {@link #setHost(String)} as the
+     * underlying code will translate the normal {@code https://storage.googleapis.com:443} into the
+     * proper Direct Path URI for you.
+     */
+    @BetaApi
+    public GrpcStorageOptions.Builder setAttemptDirectPath(boolean attemptDirectPath) {
+      this.attemptDirectPath = attemptDirectPath;
```

Comment:
Should this be enabled by default?

Reply:
With the latest version of gax-java an environment variable still needs to be set in order for direct path to actually be attempted. I think this type of thing we can change the default on in the future without there being an issue as long as there isn't any overhead to the attempt.